integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/service/ConversationController.java

```diff
@@ -82,6 +84,7 @@ public ResponseEntity statuses(
             @ApiParam(value = "Get all statuses with given convId")
             @RequestParam(value = "convId", required = false) Integer convId) {
 
+        // TODO: Add direction on status
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)